contracts/src/Plaza.sol

```diff
@@ -7,24 +7,22 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 
 contract Plaza is ERC20, Ownable, ReentrancyGuard {
-    enum ProjectType { VOLUNTEER_HOURS, FUNDRAISING }
     enum ProjectStatus { ACTIVE, COMPLETED, CANCELLED }
 
-    // Project metadata as individual public variables
+    uint256 public constant PROTOCOL_FEE_PERCENTAGE = 3;
```

Comment:
If we use `PRECISION` as a denominator, the a 3% fee would have a numerator equal to `30000`